openseattle/open-seattle-website
# feat: Implement MUI theme with basic styles

## Description

Initial MUI Theme with styles for palette, typography and button component. 

## Related Issue

[Set up primary theme for components](https://app.asana.com/0/0/1204566664333014/f)

## Acceptance Criteria

- [x] Install MUI
- [x] Add theme file
- [x] Wrap root in theme provider
- [x] Provide initial styling for project palette, typography, and button

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
| ✓ | :link: Update dependencies |
|     | :scroll: Docs              |

## Testing Steps / QA Criteria

- git pull MuiTemplate
- npm i
- test to ensure typography, palette and buttons are working:
- in volunteer.jsx import mui button and typography `import { Button, Typography } from '@mui/material'
`
- within the return statement include the following code:
```
return (
    <Layout>
      <Container>
        <Button variant='contained'>
          Volunteer With Us
        </Button>
        <Typography color={'primary.light'} variant="h1">Volunteer</Typography>
        <Typography variant="displayLarge">Volunteer</Typography>
        <Typography variant="headlineLarge">Volunteer</Typography>
        <Typography variant="titleLarge">Volunteer</Typography>
        <Typography variant="labelLarge">Volunteer</Typography>
        <Typography color={'primary'} variant="bodyLarge">Volunteer</Typography>
```
